README.md

```diff
@@ -1,7 +1,30 @@
 # Help Others
 
-To start, run npm install.
+### Start 
+1. To start, run `npm install` on the backend folder and again on the frontend folder.
+  ```
+  ; cd backend
+  ; npm install
+  ; cd frontend
+  ; npm install
+  ```
+## To run the application 
+2. Start by running the backend server `npm run dev`
+  ```
+  ; cd backend
+  ; npm run dev
+  ```
+3. Then cd into the frontend folder and run `npm start`
+  ```
+  ; cd frontend
+  ; npm run start
```

Comment:
Line 17 and 20 have different instructions - should 'run' be included?

Reply:
Yes you're absolutely right! Will action that